sim/druid/mangle.go

```diff
@@ -35,22 +45,33 @@ func (druid *Druid) registerMangleBearSpell() {
 			IgnoreHaste: true,
 			CD: core.Cooldown{
 				Timer:    druid.NewTimer(),
-				Duration: time.Duration(float64(time.Second) * (6 - durReduction)),
+				Duration: time.Second * 6,
 			},
 		},
+		// TODO: Berserk 3 target mangle cleave - Saeyon
 
-		DamageMultiplier: (1 + 0.1*float64(druid.Talents.SavageFury)) * 1.15
-		ThreatMultiplier: core.TernaryFloat64(druid.HasSetBonus(ItemSetThunderheartHarness, 2), 1.15, 1),
+		DamageMultiplier: 1.6 * (1 + 0.1*float64(druid.Talents.SavageFury)) * idolMultiplier,
+		ThreatMultiplier: 1.5,
 
 		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
-			baseDamage := 299/1.15 +
-				spell.Unit.MHWeaponDamage(sim, spell.MeleeAttackPower()) +
-				spell.BonusWeaponDamage()
+			baseDamage := spell.Unit.MHWeaponDamage(sim, spell.MeleeAttackPower())
+			dotBonusCrit := 0.0
+			if druid.LacerateBleed.Dot(target).GetStacks() > 0 {
+				dotBonusCrit = druid.FuryOfStormrageCritRatingBonus
+			}
 
+			spell.BonusCritRating += dotBonusCrit
 			result := spell.CalcAndDealDamage(sim, target, baseDamage, spell.OutcomeMeleeSpecialHitAndCrit)
+			spell.BonusCritRating -= dotBonusCrit
 
 			if result.Landed() {
 				mangleAuras.Get(target).Activate(sim)
+				if stacks := int32(druid.GetStat(stats.Defense)); stacks > 0 {
```

Comment:
I'm assuming that stats.Defense here is +Defense you have, and not your literal Defense Stat? Since the in-game defense stat is usually 300 baseline and Defender's Resolve looks at Defense > 300.

Just want to make sure we're not adding 1200 extra AP, lol.

Reply:
Ra informed me that stats.Defense is your def - 300